kt/godot-library/src/main/kotlin/godot/gen/godot/core/LambdaCallables.kt

```diff
@@ -25,27 +27,30 @@ public class LambdaCallable0<R> @PublishedApi internal constructor(
     @JvmStatic
     @JvmName("create")
     public fun <R> javaCreate(returnClass: Class<R>, function: () -> R) =
-        LambdaCallable0(variantMapper.getOrDefault(Reflection.getOrCreateKotlinClass(returnClass), NIL), function)
+        LambdaCallable0(variantMapper.getOrDefault(Reflection.getOrCreateKotlinClass(returnClass), NIL), null, function)
   }
 }
 
-public inline fun <reified R> callable0(noinline function: () -> R) =
```

Comment:
I think the onCancelCall should have a default to null, so users are not forces to write it

Reply:
done